recipes/openssl/3.x.x/conanfile.py

```diff
@@ -515,6 +516,10 @@ def _make(self):
                 # When `--prefix=/`, the scripts derive `\` without escaping, which
                 # causes issues on Windows
                 replace_in_file(self, "Makefile", "INSTALLTOP_dir=\\", "INSTALLTOP_dir=\\\\")
+                if Version(self.version) >= "3.3.0":
+                    # replace backslashes in paths with forward slashes
+                    mkinstallvars_pl = os.path.join(self.source_folder, "util", "mkinstallvars.pl")
+                    replace_in_file(self, mkinstallvars_pl, "$ENV{$k} = $v;", """$v =~ s|\\\\|/|g; $ENV{$k} = $v;""")
```

Comment:
This fixes the following error:

```
Microsoft (R) Program Maintenance Utility Version 14.29.30148.0
Copyright (C) Microsoft Corporation.  All rights reserved.

	"C:/J/workspace/prod-v2/bsr/65821/bcebd/p/straw898fb56546adc/p/bin/perl.exe" "util\mkinstallvars.pl" PREFIX=. BINDIR=apps LIBDIR= INCLUDEDIR=include APPLINKDIR=ms ENGINESDIR=engines MODULESDIR=providers "VERSION=3.3.1" "LDLIBS=C:/J/workspace/prod-v2/bsr/65821/bcebd/p/zlibbe5a5af71d5cc/p/lib/zlib.lib ws2_32.lib gdi32.lib advapi32.lib crypt32.lib user32.lib " > builddata.pm
	"C:/J/workspace/prod-v2/bsr/65821/bcebd/p/straw898fb56546adc/p/bin/perl.exe" "-I." "-Mconfigdata" "-Mbuilddata" "util\dofile.pl" "-omakefile" "exporters\cmake\OpenSSLConfig.cmake.in" > OpenSSLConfig.cmake
	"C:/J/workspace/prod-v2/bsr/65821/bcebd/p/straw898fb56546adc/p/bin/perl.exe" "-I." "-Mconfigdata" "-Mconfigdata" "-Mbuilddata" "util\dofile.pl" "-omakefile" "exporters\cmake\OpenSSLConfigVersion.cmake.in" > OpenSSLConfigVersion.cmake
	"C:/J/workspace/prod-v2/bsr/65821/bcebd/p/straw898fb56546adc/p/bin/perl.exe" "-I." "-Iutil\perl" "-Mconfigdata" "-MOpenSSL::paramnames" "util\dofile.pl" "-omakefile" "crypto\params_idx.c.in" > crypto\params_idx.c
	"C:/J/workspace/prod-v2/bsr/65821/bcebd/p/straw898fb56546adc/p/bin/perl.exe" "util\mkinstallvars.pl" "PREFIX=\\" BINDIR=bin "LIBDIR=lib" INCLUDEDIR=include APPLINKDIR=include/openssl "ENGINESDIR=\lib\engines-3" "MODULESDIR=\lib\ossl-modules" "PKGCONFIGDIR=" "CMAKECONFIGDIR=\\\lib\cmake\OpenSSL" "LDLIBS=C:/J/workspace/prod-v2/bsr/65821/bcebd/p/zlibbe5a5af71d5cc/p/lib/zlib.lib ws2_32.lib gdi32.lib advapi32.lib crypt32.lib user32.lib " "VERSION=3.3.1" > installdata.pm
	"C:/J/workspace/prod-v2/bsr/65821/bcebd/p/straw898fb56546adc/p/bin/perl.exe" "-I." "-Mconfigdata" "-Minstalldata" "util\dofile.pl" "-omakefile" "exporters\cmake\OpenSSLConfig.cmake.in" > exporters\OpenSSLConfig.cmake
Bareword found where operator expected at installdata.pm line 19, near "our $BINDIR             = 'C"
  (Might be a runaway multi-line '' string starting on line 18)
	(Do you need to predeclare our?)
```

The `"PREFIX=/"` when calling `mkinstallvars.pl` ends up generating a file where `/` is resolved as `C:\` and then the backslashes are not escaped,  resulting in a malformed generated `installdata.pm`. So this just replaces the slashes with forward ones. Unsure why it worked before - we are also not writing in those locations anyway (that is, we shouldn't end up with files written in C:/)

Reply:
Thank you so much @jcar87!